collections/_posts/2023-11-09-Elongated-structures.md

```diff
@@ -0,0 +1,355 @@
+- Create a target function that CNN can well aproximate
+- Convert the curve into a heatmap through a convolution with a filter function  
+- Linear time complexity 
+- Applicalble to various surgical 2D and 3D data task
+
+# Introduction
+
+**Elongated structures** = line, boundary, ridge or contour that should be detected in an image
+
+Detecting elongated structures in images is valuable in many fields, like geography, expression recognition, medical imaging (e.g., X-ray catheter monitoring).
+
+Traditionally, methods use differences in intensity or color to find these structures. In cases where there's no specific model for the structure, Convolutional Neural Networks (CNN) are effective.However, there hasn't been a standard method for representing these elongated structures.
+
+Two common approaches have limitations : 
+- Binary segmentation often results in gaps and fragmentation. 
+- Skeletonization is sensitive to minor errors.
+
+A new approach, used for facial boundaries detection, involves heatmaps to localise landmark. In this paper they extend the concept of heatmaps from single points to continuous curves in space, while taking in account their positional uncertainty. 
+
+![](/collections/images/elongated_structures/fig_1.jpg)
+
+# Methods
+
+## Heatmap for elongated structures
+ 
+**Representation of a curve as the image of a parameter function** :  
+
+The position of a curve $$\tilde{c} = c(t)+ z(t)$$
+with the parameter variable $$t ∈[\underset{0}{t}, \underset{n}{t}]$$ is the sum of the position of the curve $$c(t)$$ superposed by a noise term $$z(t)$$ with probability density function (pdf)
+
+- traces a noisy image of the curve
+- can be used as an optimization target for CNN training 
+
+**Description of a curve as a level set**
+
+$$f(x)$$ is a measure for the distance of a point x to the curve : \\
+$$(1) \quad C = \{x \in \mathbb{R}^n : f(x) = 0\}$$ 
+
+$$D$$ is the distribtion of a curve : \\
+$$(2) \quad D(x) = \delta(f(x))$$ 
+- with $$f(x)$$ the distance function, in the space $$\mathbb{R}^n$$
+- with $$\delta(s)$$, $$s \in \mathbb{R}$$ the Dirac function. 
+$$\delta(t) = \begin{cases}
+    0 & \text{pour } t \neq 0 \\
+    \infty & \text{pour } t = 0
+\end{cases}$$
+
+Now it is possible to convolve this distribution with a filter function $$w(x)$$ and obtain the heatmap :
+$$(3) \quad \begin{align*}
+H(x) &= D(x) * w(x) \\
+     &= \int_{\mathbb{R}^2} \delta(f(y)) w(x - y) dy \\
+     &= \int_C w(x - y) dy
+\end{align*}$$
+
+To be faster than while using a numerical integration and to generalize the description, $$H(x)$$ is simlified. They make the assumption that $$w(x)$$ is symetric $$w(x) = w(y)$$ for all $$\vert \vert x \vert \vert \underset{2}{} = \vert \vert y \vert \vert \underset{2}{}$$.
+
+If $$C$$ is an infinite straight line, the evaluation of the integral simplifies to $$h : \mathbb{R} \rightarrow \mathbb{R}$$ which only depends on the distance to the line C : 
+
+$$(4) \quad H(x)= h(f(x))$$
```

Comment:
I'm not so sure what is h(.) as it has not been defined before

Reply:
just above "the evaluation of the integral simplifies to h"